src/python/install.sh

```diff
@@ -304,7 +304,7 @@ sudo_if() {
     if [ "$(id -u)" -eq 0 ] && [ "$USERNAME" != "root" ]; then
         su - "$USERNAME" -c "$COMMAND"
     else
-        "$COMMAND"
+        $COMMAND
     fi
```

Comment:
Can we fix the `jupyterlab-git` check failures? We'd need to update 👇  to `check "jupyterlab_git" grep jupyterlab_git <<< "$packages" `.

https://github.com/devcontainers/features/blob/1f83ebc28c0b7966726151d8712c785c5c5f4f99/test/python/install_jupyterlab.sh#L20

Same change to https://github.com/devcontainers/features/blob/1f83ebc28c0b7966726151d8712c785c5c5f4f99/test/python/install_additional_jupyterlab.sh#L20

Reply:
Fixed in https://github.com/devcontainers/features/pull/694/commits/51bec72fd3e2c89d825e831c0c12fb3325d47e12